js/application.js

```diff
@@ -22,71 +24,147 @@ $(document).ready(function(){
   }
 
   var reset = function(){
-    for(i=1; i<=4; i++){
-      var resetGridRow="grid_"+i;
-      for (j=1; j<=4; j++){
-        var resetGridCol="column"+j;
-        $('#'+resetGridRow).children('#'+resetGridCol).attr('myAttr','canMove');
-        $('#'+resetGridRow).children('#'+resetGridCol).attr('combine','canCombine');
-      }
-    }
+    $('.grid').children().children().attr('myAttr', 'canMove');
+    $('.grid').children().children().attr('combine', 'canCombine');
```

Comment:
`$('.grid').children().children().attr('combine', 'canCombine');`

A better way to say this is

`$('.grid').children().children().attr('canCombine', 'true');`
`$('.grid').children().children().attr('canCombine', 'false`);`